swagger.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+swagger generate spec -o assets/swaggerui/swagger.json --scan-models -b github.com/ribice/gorsk/pkg/api
```

Comment:
`swagger generate spec -o ./swagger.json --scan-models`

This one works fine for me (and is shorter).

Reply:
cd $GOPATH/src/github.com/ribice/gorsk

My go-swagger will generate empty swagger.json without specify base directory (-b)

Reply:
I run it inside `cmd/api`, not `pkg/api`. Please update your PR to include`swagger.json`, then I can merge it. Thanks!

Reply:
Do you mean putting an extra swagger.json in `cmd/api/swagger.json`  besides `assets/swaggerui/swagger.json`?